integration/Jenkinsfiles/buildonly-daily.groovy

```diff
@@ -0,0 +1,83 @@
+
+properties([
+    [$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', daysToKeepStr: '60', numToKeepStr: '60', artifactNumToKeepStr: '1']],
+    disableConcurrentBuilds(),
+    [$class: 'GithubProjectProperty', displayName: '', projectUrlStr: 'https://github.com/nuxeo/'],
+    [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
+    parameters([booleanParam(defaultValue: true, description: '', name: 'CLEAN_REPO')]),
+    pipelineTriggers([cron('0 0 * * *')])
+])
+
+
+
+timestamps {
+    
+    timeout(time: 240, unit: 'MINUTES') {
+        
+        node ('SLAVE') {
+      
+            stage ('Checkout') {
+                checkout([$class: 'GitSCM', branches: [[name: 'master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: '', url: 'https://github.com/nuxeo/nuxeo/']]])
+            }
+            stage ('Build') {
+            
+                withEnv(["MAVEN_OPTS=-Xmx2048m"]) {
+            
+                    // Shell Clean Workspace step
+                    sh """
+                        export PATH="/opt/build/tools/maven3/bin:$PATH"
+                        [ "$CLEAN_REPO" = "true" ] && rm -rf ${WORKSPACE}/.repository/org/nuxeo 2>/dev/null
+                        ./clone.py master 
+                       """        
+                    // Maven build step
+                    withMaven(jdk: 'java-11-openjdk', maven: 'maven-3', mavenOpts: '-Xms2g -Xmx4g', mavenLocalRepo: "$WORKSPACE/.repository") {
```

Comment:
mavenOpts is redundant (and not consistent) with `withEnv(["MAVEN_OPTS`
I don't know the current status but AFAIK, mavenOpts does not work well